compose.app.yaml

```diff
@@ -0,0 +1,81 @@
+services:
+  backend:
+    build:
+      context: .
+      args:
+        - ENV=docker
+        - APP_VERSION=${APP_VERSION}
+        - APP_GIT_COMMIT=${APP_GIT_COMMIT}
+        - APP_GIT_BRANCH=${APP_GIT_BRANCH}
+        - APP_BUILD_TIME=${APP_BUILD_TIME}
+    depends_on: 
+      db:
+        condition: service_healthy
+      storage:
+        condition: service_healthy
+      migrations:
+        condition: service_completed_successfully
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:3000/health"]
+      interval: 5s
+      timeout: 5s
+      retries: 10
+    environment:
+      - ENV=docker
+      - GE_SERVER_APP_URL=http://localhost:3000
+      - GE_SERVER_PORT=3000
+      - GE_SERVER_DATABASE_HOST=db
+      - GE_SERVER_DATABASE_PORT=5432
+      - GE_SERVER_DATABASE_TIMEOUT=30s
+      - GE_SERVER_DATABASE_NAME=${POSTGRES_DB}
+      - GE_SERVER_DATABASE_USER=${POSTGRES_USER}
+      - GE_SERVER_DATABASE_PASSWORD=${POSTGRES_PASSWORD}
+
+      - GE_AUTH_ENABLE=true
+      - GE_AUTH_OIDC_PROVIDER_BASE_URL=http://keycloak:8080
+      - GE_AUTH_OIDC_PROVIDER_DOMAIN_NAME=green-ecolution
+      - GE_AUTH_OIDC_PROVIDER_AUTH_URL=http://auth.localhost:3000/realms/green-ecolution/protocol/openid-connect/auth
+      - GE_AUTH_OIDC_PROVIDER_TOKEN_URL=http://auth.localhost:3000/realms/green-ecolution/protocol/openid-connect/token
+      - GE_AUTH_OIDC_PROVIDER_PUBLIC_KEY_STATIC=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAsr2iJerMHstjk88ffPAnqwJ8owifZu1iFzdGeruYU2COAYaIANzj0COEAjEu7qiEL9wusA7QI/N/bBIOiRTJIbiIFF/wv6Jn3xUc0CdZTTTDpeYthDLJQJCdokSW4bBSxtx2T4iONGMwC51Hle7COhJ+ueZnT6aPefVpk4Nel4PmCpAOyy7vvVvwavmXcXKnZmxscF9shThGtzDf1Uq5bocO0QhYQxOy+Gz+ngcqQvCC6bZACJrTru1pDvk66EVYahRBu6cOwYtv2otyP69+LxjEFo+5wlZ6x4oc3kg8Ls48qaJg6POUvNHli/2SZk/l5uTrmDeOIvWUZsYpJB/TBQIDAQAB
```

Comment:
can be removed

Reply:
Sorry, this needs to be there until we download the keys on runtime from keycloak.